### ISSUE TITLE
How to access meta prop from search/recent (v2)?

### ISSUE THREAD

**Issue by MauricioRobayo** [PLhery/node-twitter-api-v2#35]
Hi! This is great work, thanks for putting it together.

Is there a way to access the `meta` property from the `search/recent` v2 API result?

```ts
const result = await twitterClient.search('something');
// @ts-ignore
const meta = result._realData.meta; // Is there a clean/direct way to access the meta property from the API results?
```

**Comment by PLhery** [comment]
I agree it would be nice to access it via a public getter :) will do

**Comment by PLhery** [comment]
Feature released in 1.2.0
